traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -65,17 +85,54 @@ open class AvailabilityViewController: UIViewController {
         }
     }
 
+    @IBAction func didSelectPreviousMonth(_ sender: Any) {
```

Comment:
This logic does not belong here. Currently this view controller is doing more than what it should. Availability view controller should only be in charge of fetching availabilities. Other view controllers should be in charge of dealing with calendar dates and selection.